pkg/db/errors.go

```diff
@@ -49,7 +49,7 @@ func newResourceVersionMismatch(gvk schema.GroupVersionKind, name string) error
 
 func translateDuplicateEntryErr(err error, gvk schema.GroupVersionKind, objName string) error {
 	if err, ok := err.(*mysql.MySQLError); ok && err.Number == 1062 { // error 1062 is a duplicate entry error
-		return newConflict(gvk, objName, err)
+		return newConflict(gvk, objName, fmt.Errorf("conflict"))
```

Comment:
Can we more closely replicate the error message of Kubernetes? Something like `object has been modified, please apply your changes to the latest version and try again`.

Reply:
Updated to that, I spent 5 minutes looking to see if they had hardcoded that string anywhere or if it was a variable we could piggyback off of but couldn't find it so I'm just updating the contents